site/static/help.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>rustc performance data</title>
+<link rel="stylesheet" type="text/css" href="perf.css">
+<link rel="alternate icon" type="image/png" href="/favicon-32x32.png">
+<link rel="icon" type="image/svg+xml" href="/favicon.svg">
+<style>
+    .help-content {
+        font-family: Helvetica, Arial, sans-serif;
+        line-height: 140%;
+        font-size: 16px;
+        max-width: 50em;
```

Comment:
This is to prevent the lines from getting too long, which can make the text very hard to read. In case you're wondering, I checked by resizing my window and it should look fine on mobile as well.